wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/importer/ProjectImportOutputJsonRpcRegistrar.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.project.server.importer;
+
+import org.eclipse.che.api.core.jsonrpc.commons.RequestHandlerConfigurator;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import java.util.Set;
+
+import static com.google.common.collect.Sets.newConcurrentHashSet;
+
+/**
+ * Endpoint registry for broadcasting project import events. Holds registered client's endpoint ids.
+ *
+ * @author Vlad Zhukovskyi
+ * @since 5.9.0
+ */
+@Singleton
+public class ProjectImportOutputJsonRpcRegistrar {
+
+    private static final String EVENT_IMPORT_OUTPUT_SUBSCRIBE    = "importProject/subscribe";
```

Comment:
Maybe it would be better to place these constants to the shared [Constants.java](https://github.com/eclipse/che/blob/master/wsagent/che-core-api-project-shared/src/main/java/org/eclipse/che/api/project/shared/Constants.java#L16)?